### PR TITLE
Responsive trashbin table

### LINKED ISSUE #1760: file names are not visible in trashbin page while browsing through mobile phone
### Steps to reproduce
Using your mobile phone:-
1. Delete some files and browse to the trashbin page.
   Here file name is not displayed
 

2. Also you have to scroll right to get restore option, which is not a good design
  

### Expected behaviour
File name must be displayed in trashbin

### Actual behaviour
File name is not properly displayed in trashbin

### LINKED ISSUE #1763: Batch Action Using Mobile Phone
### Steps to reproduce
1. Select all files for batch action in files page or trashbin page. Here ```Name```, ```Search```, ```+New``` .... all overlap with each other. Also once all file have been selected for batch action, we can't unselect all file in batch as ```all file selection checkbox``` is overridden by blue colored ```+New``` box as in screenshot below:

2. If you perform batch action in trashbin page, whole page is filled with success and failure messages

### Expected behaviour
- It must be possible to batch unselect file after batch selection
- ```Name``` , ```Search``` rows should not overlap with each other while performing batch selection
- Success and failure message should not occupy whole OC screen after batch action

### Actual behaviour
- It is not possible to batch unselect file after batch selection
- ```Name``` , ```Search``` rows  overlap with each other while performing batch selection
- Success and failure messages occupy whole OC screen after batch action

### PR DESCRIPTION
**Pull request by Julian1998**
## Description

Made the trashbin table responsive. (the same way the files list looked like)

## Related Issue

- Fixes #1760 

## How Has This Been Tested?

- delete a file
- check it on a mobile phone

## Screenshots (if appropriate):
Bildschirmfoto von »2019-10-17 21-35-56«
Bildschirmfoto von »2019-10-17 21-36-06«

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by Julian1998** [comment]
@LukasHirt The appbar seems to be a general problem as well as the uikit notifications (according to #1763). Should be done in a new PR/Issue

**Comment by Julian1998** [comment]
The topbar/appbar view is a general problem as you can see in #1763

**Comment by Julian1998** [comment]
@LukasHirt please review :)

File: apps/files/src/components/Trashbin.vue [hunk 1]
```diff
@@ -16,28 +16,54 @@
           <oc-table-cell>
             <oc-checkbox class="uk-margin-small-left" @change.native="$_ocTrashbin_toggleFileSelect(item)" :value="selectedFiles.indexOf(item) >= 0" />
           </oc-table-cell>
-          <oc-table-cell class="uk-text-truncate">
```
Comment (LukasHirt) [inline apps/files/src/components/Trashbin.vue]: Pls, keep this here - if the file name is too long we need to truncate it.

File: apps/files/src/components/Trashbin.vue [hunk 2]
```diff
@@ -16,28 +16,54 @@
...
             </oc-button>
-            <oc-button icon="delete" @click="$_ocTrashbin_deleteFile(item)" ariaLabel="Delete">
+            <oc-button icon="delete" class="uk-visible@m" @click="$_ocTrashbin_deleteFile(item)" ariaLabel="Delete">
               <translate>Delete immediately</translate>
             </oc-button>
+            <oc-button
+              :id="'files-trashbin-file-list-action-button-small-resolution-' + index"
+              icon="more_vert"
+              class="uk-hidden@m"
+              :aria-label="'show-file-actions'"
```
Comment (LukasHirt) [inline apps/files/src/components/Trashbin.vue]: Needs to be translated.

**Comment by Julian1998** [comment]
@LukasHirt Applied your recquested changes! :)

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>XGAPortrait</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6110/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>iPhone</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6110/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>iPhone</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6114/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>XGAPortrait</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6114/

<p>















</p></details>

**Comment by PVince81** [comment]
@Julian1998 if you need another review after adjusting, please always ping to ask as we can't guess if this is still work in progress

**Comment by Julian1998** [comment]
@PVince81 I see. I thought re-requesting a review would be enough.

So @PVince81 @LukasHirt please review

**Pull request merged**